dev-docs/publisher-api-reference.md

```diff
@@ -1290,4 +1280,77 @@ This call returns the following DFP video ad tag URL:
 https://pubads.g.doubleclick.net/gampad/ads?env=vp&gdfp_req=1&output=vast&unviewed_position_start=1&correlator=1507127916397&sz=640x480&url=http://www.referer-url.com&iu=/19968336/prebid_cache_video_adunit&cust_params=hb_bidder%3DappnexusAst%26hb_adid%3D26d4996ee83709%26hb_pb%3D10.00%26hb_size%3D640x480%26hb_uuid%3D16c887cf-9986-4cb2-a02f-8e9bd025f875%26section%3Dblog%26anotherKey%3DanotherValue&hl=en
 ```
 
+In this example, we are using `options.url`  to pass in the video ad server directly:
+
+```javascript
+var adserverTag = 'https://pubads.g.doubleclick.net/gampad/ads?'
++ 'sz=640x480&iu=/19968336/prebid_video_adunit&impl=s&gdfp_req=1'
++ '&env=vp&output=xml_vast2&correlator=' + Date.now();
+
+var videoUrl = pbjs.adServers.dfp.buildVideoUrl({
+    adUnit: videoAdUnit,
+    url: adserverTag
+});
+```
+
 </div>
+
+<a name="buildVideoUrl-options-object" />
+
+#### The `options` object
+
+The `options` object has the following fields:
+
+{: .table .table-bordered .table-striped }
+| Field  | Type   | Description                                                                                                                                                                         |
+|--------+--------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| adUnit | object | (Required) The Prebid adUnit to which the returned URL will map.                                                                                                                    |
+| bid    | object | (Optional) The Prebid bid for which targeting will be set. If this is not defined, Prebid will use the bid with the highest CPM for the adUnit.                                     |
+| params | object | (Required) Querystring parameters that will be used to construct the DFP video ad tag URL.  Publisher-supplied values will override values set by Prebid.js.  See below for fields. |
+| url    | string | (Optional) The video ad server URL.  When given alongside `params`, the parsed URL will be overwritten with any matching components of `params`.                                    |
+
+<a name="buildVideoUrl-options-params" />
+
+#### The `options.params` object
+
+The `options.params` object contains the parameters needed to form a URL which hits the [DFP API](https://support.google.com/dfp_premium/answer/1068325?hl=en).  It has the following fields:
+
+<span style="color: rgb(255,0,0);">FIXME</span>: is "arbitraryKey" still supported?  not seeing anything explicitly mentioning it in the code at a glance, but may have missed it.  It's also not mentioned in the JSDoc.
```

Comment:
As far as I can tell, no. I hadn't seen usage of this before, and it seems like sending custom key-value pairs is covered by the `cust_params` field, so possibly `cust_params` is a replacement for that